src/PrestaShopBundle/Controller/Admin/Sell/Catalog/AttributeGroupController.php

```diff
@@ -80,12 +83,37 @@ public function indexAction(Request $request, AttributeGroupFilters $attributeGr
      *     message="You do not have permission to create this."
      * )
      *
-     * @return RedirectResponse
+     * @param Request $request
+     *
+     * @return Response
      */
-    public function createAction()
+    public function createAction(Request $request): Response
     {
-        //@todo: implement in antoher pr
-        return $this->redirectToRoute('admin_attribute_groups_index');
+        $attributeGroupFormBuilder = $this->get('PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Builder\AttributeGroupFormBuilder');
+        $attributeFormHandler = $this->get('PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Handler\AttributeGroupFormHandler');
+
+        $attributeGroupForm = $attributeGroupFormBuilder->getForm();
+        $attributeGroupForm->handleRequest($request);
+
+        try {
+            $handlerResult = $attributeFormHandler->handle($attributeGroupForm);
+
+            if (null !== $handlerResult->getIdentifiableObjectId()) {
+                $this->addFlash('success', $this->trans('Successful creation', 'Admin.Notifications.Success'));
+
+                return $this->redirectToRoute('admin_attribute_groups_index');
+            }
+        } catch (Exception $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages()));
+        }
+
+        return $this->render(
+            '@PrestaShop/Admin/Sell/Catalog/AttributeGroup/create.html.twig',
+            [
+                'layoutTitle' => $this->trans('New attribute group', 'Admin.Navigation.Menu'),
```

Comment:
Attribute is another entity.
There are AttributeGroup (`size,color` etc.) and Attribute (green,XL,L etc.). This Pr is about the group.
It would probably make less confusion if we have named them similarly as features (`attribute` and `attribute value` respectively), but it is already done the way it is.

Reply:
@zuk3975 @Hlavtox Hello, could I get more context? If you have a screenshot of said wording in PS it would be nice. Thanks

Reply:
@zuk3975 On legacy page, it is called an `attribute` and a `value`, it should be kept. 👍

![Snímek obrazovky 2023-04-12 161332](https://user-images.githubusercontent.com/6097524/231485253-a4daa02a-5256-4bc8-a7a4-8e9a6b9dcea1.jpg)

Reply:
It's wrong only here:

![Snímek obrazovky 2023-04-12 161435](https://user-images.githubusercontent.com/6097524/231485538-9222f0c9-6ded-498c-8a91-25be19b1500a.jpg)

Reply:
Developer documentation follows the same wording as I say:
- https://build.prestashop-project.org/prestashop-specs/1.7/back-office/catalog/attributes-and-features/attributes/add-edit-attribute.html
- https://build.prestashop-project.org/prestashop-specs/1.7/back-office/catalog/attributes-and-features/attributes/add-edit-attribute-value.html

User documentation follows the same wording as I say:
- https://docs.prestashop-project.org/1.7-documentation/user-guide/selling/managing-catalog/managing-product-attributes

> but it is already done the way it is.

Then it will need to be remade. 🤷‍♂️